src/Opdex.Auth.Api/Controllers/AuthController.cs

```diff
@@ -34,7 +34,7 @@ public AuthController(IMediator mediator, IJwtIssuer jwtIssuer, IOptionsSnapshot
     [HttpGet("authorize")]
     public async Task<IActionResult> Authorize([FromQuery] AuthorizeRequestQuery query, CancellationToken cancellationToken)
     {
-        var session = new AuthSession(query.CodeChallenge, query.CodeChallengeMethod);
+        var session = new AuthSession(new Uri(query.RedirectUri), query.CodeChallenge, query.CodeChallengeMethod);
```

Comment:
Does it matter that this can be a very specific URL? 
`https://app.opdex.com` vs `https://app.opdex.com/auth?something=123`